### DISCUSSION TITLE
create-next-app not prompting

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [vercel/next.js#54622] [Help] [answered]
### Summary

When I run `npx create-next-app@latest` the only prompt I get the following:

```
What is your project named? my-app
```

But it doesn't ask me the rest of the questions

```
Would you like to use TypeScript? No / Yes
Would you like to use ESLint? No / Yes
Would you like to use Tailwind CSS? No / Yes
Would you like to use `src/` directory? No / Yes
Would you like to use App Router? (recommended) No / Yes
Would you like to customize the default import alias? No / Yes
What import alias would you like configured? @/*
```

**Comment by zaaakher** [comment, marked as answer]
The older versions of `create-next-app` only asks you for a project name.
Make sure the `create-next-app` npm package in your machine is the [latest version](https://www.npmjs.com/package/create-next-app)

You can run `npm ls -g` to see the global packages and scroll around to see the version of `create-next-app`

Or you can run the following command 
 ```
npm view create-next-app version
```

**Reply by TheMikeyRoss** [reply]
Thanks mate
